drv.go

```diff
@@ -5,30 +5,30 @@
 
 // Package godror is a database/sql/driver for Oracle DB.
 //
-// The connection string for the sql.Open("godror", connString) call can be
+// The connection string for the sql.Open("godror", connstr) call can be
```

Comment:
This should say "dataSourceName" instead of connstr. Below instead of sid it should say connectString.